src/main/kotlin/com/spectralogic/bp/bench/cli/WriteToTapeCommand.kt

```diff
@@ -3,15 +3,115 @@
  *   Copyright 2014-2019 Spectra Logic Corporation. All Rights Reserved.
  * ***************************************************************************
  */
+/*
+ * ****************************************************************************
+ *   Copyright 2014-2019 Spectra Logic Corporation. All Rights Reserved.
+ * ***************************************************************************
+ */
  
 package com.spectralogic.bp.bench.cli
 
 import com.github.ajalt.clikt.core.CliktCommand
-import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.output.TermUi
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.prompt
+import com.github.ajalt.clikt.parameters.options.validate
+import com.github.ajalt.clikt.parameters.types.choice
+import com.github.ajalt.clikt.parameters.types.int
+import com.github.ajalt.clikt.parameters.types.long
+import com.spectralogic.ds3client.Ds3ClientBuilder
+import com.spectralogic.ds3client.commands.HeadBucketRequest
+import com.spectralogic.ds3client.commands.HeadBucketResponse
+import com.spectralogic.ds3client.commands.spectrads3.PutBucketSpectraS3Request
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers
+import com.spectralogic.ds3client.models.bulk.Ds3Object
+import com.spectralogic.ds3client.models.common.Credentials
+import com.spectralogic.ds3client.networking.FailedRequestException
+
+class WriteToTapeCommand : CliktCommand(name = "put") {
+    companion object {
+        val sizeMap = mapOf<String, Long>(
+            Pair("b", 1L),
+            Pair("Kb", 3L),
+            Pair("Mb", 6L),
+            Pair("Gb", 9L),
+            Pair("Tb", 12L)
+        )
+    }
+
+    private val endpoint by option(
```

Comment:
Similar question about the data type here.

Reply:
@rpmoore would it be better to store it as a URI?

Reply:
Since the SDK takes a String let's just rely on it's validation logic.  We may want to give an example of what it looks like in the help doc.

Also is there a way to pull the common options out?  Don't want to repeat ourselves for the args.

Reply:
I'll make those two changes and re-submit